docs/supported/index.rst

```diff
@@ -188,17 +188,16 @@ Supported Operating Systems
 	        <a href="../index.html" class="button">Learn more</a>
 	    </div>
 	    <div class="pod-boxout pod-boxout--4up pod-boxout--orchestration">
+			<img src="../_static/images/rhel72x.png" alt="RHEL 7 logo"/>
+			<span>RHEL 7</span>
+	        <a href="../index.html" class="button">Learn more</a>
```

Comment:
I think it's because the CoreOS installation instructions are off on our blog somewhere while the other "traditional" OS installations all require the user to choose their orchestration framework first.